### DISCUSSION TITLE
Does AbracaDABra delete entries when a different multiplex if found on the same channel?

### DISCUSSION THREAD

**Discussion by theJazzman42** [KejPi/AbracaDABra#83] [General]
It looks to me like the old entry is being deleted. I received on 11C both S-ANHALT as well as Polskie Radio in the last days. Today there  is just Polskie Radio on my list. Even though I miss a third entry for Polskie Radio (11A), while I didn't receive any another multiplex on this channel... Hmm... <scratching_my_head>.

![image](https://github.com/KejPi/AbracaDABra/assets/123947155/da557367-d854-49d1-b878-7cc6dd68e16b)

**Comment by KejPi** [comment]
Yes, when new ensemble is received in the channel, the old one is removed and replaced by new. This is also valid when services are added or removed to the same ensemble while the application is not tuned to that channel.

**Comment by andimik** [comment]
> I received on 11C both S-ANHALT as well as Polskie Radio in the last days. Today there is just Polskie Radio on my list. 

Then you cannot receive Sachsen-Anhalt any more. That's why the app is replacing it by the Polish multiplex.

For me, this behaviour is much more practical for the user than in other tools.
